osquery/events/darwin/fsevents.cpp

```diff
@@ -165,6 +165,9 @@ void FSEventsEventPublisher::Callback(
 bool FSEventsEventPublisher::shouldFire(
     const FSEventsSubscriptionContextRef& mc,
     const FSEventsEventContextRef& ec) const {
+  // This is stopping us from getting events on links
+  // If we need this feature later, this code will
+  // Have to be updated
```

Comment:
Just the way I wrote comments mentally keeping the 80 characters in line. I've updated the comment